test/bin/common.sh

```diff
@@ -21,6 +21,10 @@ INSTALLER_IMAGE_BLUEPRINT="rhel-9.2"
 # The location the web server should serve.
 export IMAGEDIR="${ROOTDIR}/_output/test-images"
 
+# The location for container list to mirror to a local registry
+# shellcheck disable=SC2034  # used elsewhere
+MIRROR_CONTAINERS_LIST="${IMAGEDIR}/registry-mirror-containers.txt"
```

Comment:
If only `mirror_registry.sh` uses this file, we don't need to put the variable in `common.sh`.

Reply:
Done